$OpenBSD: patch-libraries_base_Makefile,v 1.1 2004/04/26 23:48:50 dons Exp $
Fix to prevent stub file building when bootstrapping.
This is a bug in the build infrastructure.

--- libraries/base/Makefile.orig	Thu Dec 18 22:28:46 2003
+++ libraries/base/Makefile	Thu Dec 18 22:28:48 2003
@@ -72,12 +72,13 @@
 CLEAN_FILES += GHC/PrimopWrappers.hs
 
 # -----------------------------------------------------------------------------
-
+ifneq "$(BootingFromHc)" "YES"
 STUBOBJS += \
    Control/Concurrent_stub.$(way_)o
    
 CLEAN_FILES += $(STUBOBJS) \
    Control/Concurrent_stub.[ch]
+endif
 
 #-----------------------------------------------------------------------------
 # 	Building the library for GHCi
